src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,74 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents the file interaction between the <code>Parser</code> object
+ * and the file stored on the hard disk of the user.
+ */
+public class Storage {
+
+    private static String filePath;
+    private File f;
+    private ArrayList<String> tasksList;
+
+    /**
+     * Initialises a <code>Storage</code> object that initialises a <code>File</code>
+     * object that will be used to read and write to. Initialises a <code>ArrayList</code>
+     * to store the tasks that was contained in the text file.
+     * @param filePath A <code>String</code> representing the file path.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+        f = new File(this.filePath);
+        tasksList = new ArrayList<>();
+    }
+
+    /**
+     * Reads the tasks from the file and loads them into an <code>ArrayList</code>
+     * @return An <code>ArrayList</code> containing the tasks from the file.
+     */
+    public ArrayList<String> load() {
```

Comment:
Could consider renaming this loadFromFile to standardise with appendToFile

Reply:
Thank you for the suggestion. I will change it soon.